src/usr/local/www/services_unbound_advanced.php

```diff
@@ -369,7 +369,7 @@
 	'disable_auto_added_access_control',
 	'Disable auto-added access control',
 	'disable the automatically-added access control entries',
-	$pconfig['hdisable_auto_added_access_control']
+	$pconfig['disable_auto_added_access_control']
```

Comment:
Need to have a rant here. What in the *** happened to code review/test in the original repo where this bootstrap stuff was done??? There are bugs like this all through the code - hundreds have been got out so far and who knows how many more there are to be found.
2.2.5 is a good piece of software, and 2.3 will be a PoC unless every line of code is re-checked to find all the dumb bugs introduced by the bootstrap rush.
End of rant